mailmerge/__main__.py

```diff
@@ -79,9 +84,15 @@
     type=click.Choice(["colorized", "text", "raw"]),
     help="Output format (colorized).",
 )
+@click.option(
```

Comment:
I think that creating a log by default is reasonable.  Maybe we can just call it `mailmerge.log`?  That would simplify the command line interface.